common.blocks/menu/menu.bh.js

```diff
@@ -31,6 +37,8 @@ module.exports = function(bh) {
                     var i = 0, itemOrGroup;
                     while(itemOrGroup = content[i++]) {
                         if(itemOrGroup.block === 'menu-item') {
+                            itemOrGroup.attrs = (itemOrGroup.attrs || {});
+                            itemOrGroup.attrs.id = itemOrGroup.attrs.id || ctx.generateId(); // Support a11y
```

Comment:
AFAI: `ctx.generateId()` returns uniq id, for every context `ctx`. If so, all menu items would have the id

//cc @mishanga